### ISSUE TITLE
Movie status and size changing while 'Refresh Movie' task in progress

### ISSUE THREAD

**Issue by psi5asp** [Radarr/Radarr#6480]
When the "Refresh Movie" task is underway, looking at a "Missing" movie, the status changes to "Downloaded" and the size keeps adjusting from 0 to (what appears to be) an accumulative number (possibly from the scan). Refreshing the page resolves it until after a few minutes and it comes back.

The movie should keep the DB status "Missing" with 0 size until the file is actually downloaded (ie goes through the correct proccess of finding, sending to client, downloading...then the status should change to "Downloading" and once downloaded THEN change to "Downloaded" with its filesize shown in the "FIles" tab, etc..

![image](https://user-images.githubusercontent.com/41772110/126628797-5c0c032e-7abc-4a5e-87ed-026c07757b0d.png)

**Platform Information (please complete the following information):**
 - OS: Debian GNU/Linux 10 (buster)
 - Docker:  No
 - Mono or .NET Version (System -> Status):  .NET 5.0.1
 - Browser and Version (Only needed for UI issues): Opera 77.0.4054.254
 - Radarr Version: 3.2.2.5080
 - Radarr Branch: master

**Trace Logs**
Turn on Trace logs under Settings -> General and wait for the bug to occur again. 
**Upload the full log file here (or another site (e.g. pastebin) and link it). Issues will be closed, if they do not include this!**

Logs show nothing related to viewed movie when task is running, so its probably some cross-variable update issue.

**Comment by austinwbest** [comment]
Yes, there is a signalr glitch at play when status changes happen (to other movies) while a movie details page is open.
